### PR TITLE
DF-LCCD

### PR DESCRIPTION
**Pull request by JonathonMisiewicz**
## Description
The next method on my list for @loriab 

## What are your new additions? Please provide a brief list.
* **New Features**
  - [x] DF-LCCD!

## Any questions for the community?
- [ ] Are there any citations worth adding? These equations are fairly simple to density-fit.

## Status
- [x] Click when ready for review-and-merge

### CODE REVIEW

File: Coupled-Electron-Pair-Approximation/DF-LCCD.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+D = Fo.reshape(-1, 1, 1, 1) + Fo.reshape(-1, 1, 1) - Fv.reshape(-1, 1) - Fv
+t2 = np.zeros(D.shape)
+dsd = DirectSumDiis(3, 8)
+nvir = len(Fv)
+
+### Main Loop
+for i in range(1, maxiter + 1):
+    # Two Electron Terms
+    # r2 = g^ij_ab
+    temp = np.einsum("iaq, jbq -> ijab", R["ov"], R["ov"], optimize = True)
```
Comment (dgasmith) [inline Coupled-Electron-Pair-Approximation/DF-LCCD.py:45]: `optimize=True`, can you run a formatter over this in general?

Please do leave spaces above comment lines as well.

Comment (JonathonMisiewicz) [reply Coupled-Electron-Pair-Approximation/DF-LCCD.py:45]: As in, every comment should have a blank line before it?

Comment (dgasmith) [reply Coupled-Electron-Pair-Approximation/DF-LCCD.py:45]: Yes, help readability I believe.

Comment (JonathonMisiewicz) [reply Coupled-Electron-Pair-Approximation/DF-LCCD.py]: Formatted with column limit 119.

**Pull request merged**